README.md

```diff
@@ -4,10 +4,17 @@
   <a href="https://github.com/actions/setup-ruby"><img alt="GitHub Actions status" src="https://github.com/actions/setup-ruby/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-This action sets up a ruby environment for use in actions by:
+This action sets up Ruby for use in actions by adding a version of Ruby already existing in the toolcache to the PATH.
 
-- optionally installing a version of ruby and adding to PATH. Note that this action only uses versions of Ruby already installed in the cache. The action will fail if no matching versions are found.
-- registering problem matchers for error output
+This action currently only supports versions of Ruby in the toolcache.
+See [this list](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md) for the available Ruby versions.
```

Comment:
https://github.com/actions/virtual-environments/blob/master/images/linux/toolcache-1804.json might be a good link too but that only list versions in `major.minor` format, which might be confusing.

Reply:
Is there a centralized place listing the Ruby version in virtual environments?

https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md is rather long (needs manual searching) and is a platform-specific README.

Reply:
https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md is the definitive list but it's not accurate everywhere and if you follow it, you will break as it rolls out replacements as it's not updated until it's everywhere and you will break at some date prior to that.

Reply:
That image list should list exactly what's currently everywhere but caution it's not safe to bind to it.  This readme should point to it and recommend binding to minor versions as the exact version slides.

Reply:
The paragraph just below mentions:
> Because versions in the toolcache are updated over time, it is necessary to use a version range such as `2.6.x` and not exact versions.

So do you think the wording here is fine or should something change in this PR?
If the later, could you make a suggestion since you know about the rollout process?

Reply:
I added a sentence about it and highlighted that a version range should be used.